haskellPackages: update stackage and hackage

maralorn opened the pull request:
### This Merge

This PR is the regular merge of the `haskell-updates` branch into `master`.

This branch is being continually built and tested by hydra at https://hydra.nixos.org/jobset/nixpkgs/haskell-updates.

I will aim to merge this PR **by 2021-06-27**. If I can merge it earlier, there might be successor PRs in that time window. As part of our rotation @cdepillabout will continue these merges from 2021-06-28 to 2021-07-11.

### haskellPackages Workflow Summary

Our workflow is currently described in
[`pkgs/development/haskell-modules/HACKING.md`](https://github.com/NixOS/nixpkgs/blob/haskell-updates/pkgs/development/haskell-modules/HACKING.md).

The short version is this:
* We regularly update the Stackage and Hackage pins on `haskell-updates` (normally at the beginning of a merge window).
* The community fixes builds of Haskell packages on that branch.
* We aim at at least one merge of `haskell-updates` into `master` every two weeks.
* We only do the merge if the [`mergeable`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/mergeable) job is succeeding on hydra.
* If a [`maintained`](https://hydra.nixos.org/job/nixpkgs/haskell-updates/maintained) package is still broken at the time of merge, we will only merge if the maintainer has been pinged 7 days in advance. (If you care about a Haskell package, become a maintainer!)

---

This is the follow-up to #125977.

sternenseemann committed:
haskell-generic-builder: allow passing flags to the test suite(s)

Every flag the generic builder receives via `testFlags` is passed via
`--test-option` [1] to `Setup.hs` which in turn passes them to the
underlying test suite binary. These wrapped options are added to
`checkFlagsArray` in `checkPhase`. This needs to be done in bash since
without structuredAttrs in nixpkgs so far, Nix arrays aren't properly
translated into bash arrays, so we'd have all sorts of quoting issues
when spaces are involved.

Re-using `checkFlags` and `checkFlagsArray` from standard stdenv
setup.sh also results in an additional feature: Using `overrideAttrs`
`checkFlags` and `checkFlagsArray` can additionally be overridden,
which allows passing extra flags to `Setup.hs` whithout being wrapped
with `--test-option`.

[1]: See also https://cabal.readthedocs.io/en/3.4/setup-commands.html?highlight=test-option#cmdoption-runhaskell-Setup.hs-test-test-option
     According to the cabal-install man page this also allows passing
     special variables which are substituted for other values
     depending on context.

maralorn committed:
haskellPackages: stackage-nightly 2021-06-01 -> 2021-06-10

This commit has been generated by maintainers/scripts/haskell/update-stackage.sh

maralorn committed:
all-cabal-hashes: 2021-06-06T15:18:11Z -> 2021-06-12T16:23:24Z

This commit has been generated by maintainers/scripts/haskell/update-hackage.sh

maralorn committed:
haskellPackages: regenerate package set based on current config

This commit has been generated by maintainers/scripts/haskell/regenerate-hackage-packages.sh

maralorn committed:
haskelPackages: HACKING.md, slight clarification

maralorn committed:
haskellPackages: HACKING.md, clarify PR title

maralorn committed:
haskellPackages: HACKING.md: Insert links into template

sternenseemann committed:
haskell.packages.ghc901.profunctors: fix eval error

Stackage has updated profunctors to 5.6.2, so the override is
unnecessary.

sternenseemann committed:
haskellPackages.ptr-poker: no longer mark as broken

sternenseemann committed:
Revert "haskellPackages.gtk2hs-buildtools: attempt to fix build on aarch64"

This reverts commit 38c4256a9635d37ed297e43ade8e6f5ef42893d1.

The attempt failed, gtk2hs-buildtools still fails with the same linker error.

sternenseemann committed:
haskellPackages.safe-coloured-text-*: mark as no longer broken

maralorn committed:
Merge pull request #126364 from sternenseemann/haskell-test-flags

haskell-generic-builder: allow passing flags to the test suite(s)

maralorn commented:
### [haskell-updates build report from hydra](https://hydra.nixos.org/jobset/nixpkgs/haskell-updates)
*evaluation [1678005](https://hydra.nixos.org/eval/1678005) of nixpkgs commit [222b6c8](https://github.com/NixOS/nixpkgs/commits/222b6c80ac00363fb53d2cad40305281b75bd7e5) as of 2021-06-13 14:49 UTC*
#### Build summary

 | Platform | Failed :x: | DependencyFailed :heavy_exclamation_mark: | TimedOut :hourglass::no_entry_sign: | Unfinished :hourglass_flowing_sand: | Success :heavy_check_mark: | 
 | --- | --- | --- | --- | --- | --- | 
 | [aarch64-linux :iphone:](https://hydra.nixos.org/eval/1678005?filter=.aarch64-linux) | 37 | 96 | 8 | 3 | 6523 | 
 | [x86_64-darwin :apple:](https://hydra.nixos.org/eval/1678005?filter=.x86_64-darwin) |  |  |  | 6617 | 3 | 
 | [x86_64-linux :penguin:](https://hydra.nixos.org/eval/1678005?filter=.x86_64-linux) | 19 | 30 | 2 | 1 | 6654 | 
#### Maintained packages with build failure
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145323762) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327314) [[:penguin::x:]](https://hydra.nixos.org/build/145315926) [haskellPackages.dhall-openapi](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.dhall-openapi) @Gabriel439
- [ ] [ghc901](https://hydra.nixos.org/eval/1678005?filter=ghc901) @kosmikus @marcweber @peti
  - [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/145324453) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145332994) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145335308) [haskell.compiler](https://hydra.nixos.org/eval/1678005?filter=haskell.compiler.ghc901)
  - [[:iphone::x:]](https://hydra.nixos.org/build/145335591) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145318196) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145330190) [haskell.compiler.integer-simple](https://hydra.nixos.org/eval/1678005?filter=haskell.compiler.integer-simple.ghc901)
- [ ] [ghcHEAD](https://hydra.nixos.org/eval/1678005?filter=ghcHEAD) @kosmikus @marcweber @peti
  - [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145324366) [[:penguin::x:]](https://hydra.nixos.org/build/145320091) [haskell.compiler](https://hydra.nixos.org/eval/1678005?filter=haskell.compiler.ghcHEAD)
  - [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145319898) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145329178) [haskell.compiler.native-bignum](https://hydra.nixos.org/eval/1678005?filter=haskell.compiler.native-bignum.ghcHEAD)
- [ ] [spago](https://hydra.nixos.org/eval/1678005?filter=spago) @cdepillabout
  - [[:iphone::x:]](https://hydra.nixos.org/build/145322873) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335230) [[:penguin::x:]](https://hydra.nixos.org/build/145316718) [toplevel](https://hydra.nixos.org/eval/1678005?filter=spago)
  - [[:iphone::x:]](https://hydra.nixos.org/build/145332146) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334185) [[:penguin::x:]](https://hydra.nixos.org/build/145317539) [haskellPackages](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.spago)
#### Maintained packages with failed dependency
- [ ] [dhall-nix](https://hydra.nixos.org/eval/1678005?filter=dhall-nix) @Gabriel439
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145317243) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145318487) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145318404) [toplevel](https://hydra.nixos.org/eval/1678005?filter=dhall-nix)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145326613) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327324) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145321160) [haskellPackages](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.dhall-nix)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145324315) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145330599) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145326281) [haskellPackages.dhall-nixpkgs](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.dhall-nixpkgs) @Gabriel439
- [ ] [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145329914) [maintained](https://hydra.nixos.org/eval/1678005?filter=maintained) @cdepillabout @maralorn @sternenseemann
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145333238) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145331463) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145318383) [nix-linter](https://hydra.nixos.org/eval/1678005?filter=nix-linter) @marsam
- [ ] [update-nix-fetchgit](https://hydra.nixos.org/eval/1678005?filter=update-nix-fetchgit) @expipiplus1 @sorki
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145316980) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145325404) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145332161) [toplevel](https://hydra.nixos.org/eval/1678005?filter=update-nix-fetchgit)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145317861) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145321112) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145325789) [haskellPackages](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.update-nix-fetchgit)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145324807) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145319065) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145327353) [haskellPackages.yarn2nix](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.yarn2nix) @sternenseemann
#### Unmaintained packages with build failure
<details><summary>39 job(s) </summary>

- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145330469) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145320535) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145318378) [haskellPackages.HsASA](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.HsASA) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145329927) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333585) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327515) [haskellPackages.OrderedBits](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.OrderedBits) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145322197) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145329186) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327254) [haskellPackages.accelerate-llvm](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.accelerate-llvm) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145317886) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334184) [[:penguin::x:]](https://hydra.nixos.org/build/145321975) [haskellPackages.amazonka-s3-streaming](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.amazonka-s3-streaming) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145322820) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145329949) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145317953) [haskellPackages.basic-cpuid](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.basic-cpuid) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/145320620) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333444) [[:penguin::x:]](https://hydra.nixos.org/build/145330769) [haskellPackages.broadcast-chan-conduit](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.broadcast-chan-conduit) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145330158) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145332726) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145322254) [haskellPackages.cdar-mBound](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.cdar-mBound) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/145319856) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145319692) [[:penguin::x:]](https://hydra.nixos.org/build/145332861) [haskellPackages.churros](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.churros) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145318650) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145322356) [[:penguin::x:]](https://hydra.nixos.org/build/145332244) [haskellPackages.composite-base](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-base) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145321488) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145317989) [[:penguin::x:]](https://hydra.nixos.org/build/145322424) [haskellPackages.cronus](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.cronus) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145324882) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145316385) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145333158) [haskellPackages.easytensor](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.easytensor) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145331130) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327068) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145330504) [haskellPackages.freetype2](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.freetype2) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145330744) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145320717) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145323687) [haskellPackages.ghc-prof](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.ghc-prof) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145323615) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145318279) [haskellPackages.gnome-keyring](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gnome-keyring) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145321031) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145331481) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145316594) [haskellPackages.gtk2hs-buildtools](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk2hs-buildtools) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145332958) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145318926) [[:penguin::x:]](https://hydra.nixos.org/build/145332492) [haskellPackages.hnix](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.hnix) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145326237) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335877) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145333572) [haskellPackages.hssh](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.hssh) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145330684) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145320581) [[:penguin::x:]](https://hydra.nixos.org/build/145321941) [haskellPackages.hw-mquery](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.hw-mquery) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/145328016) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145322947) [[:penguin::x:]](https://hydra.nixos.org/build/145334252) [haskellPackages.immortal-queue](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.immortal-queue) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145317549) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145329035) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145321644) [haskellPackages.inline-r](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.inline-r) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145321983) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145323991) [[:penguin::x:]](https://hydra.nixos.org/build/145326213) [haskellPackages.kure](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.kure) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145328534) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334309) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145334497) [haskellPackages.libBF](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.libBF) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145327514) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333122) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145323457) [haskellPackages.linear-accelerate](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.linear-accelerate) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145333194) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145321238) [[:penguin::x:]](https://hydra.nixos.org/build/145328305) [haskellPackages.list-tries](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.list-tries) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145331333) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333832) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145323332) [haskellPackages.long-double](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.long-double) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/145327464) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145330877) [[:penguin::x:]](https://hydra.nixos.org/build/145327988) [haskellPackages.mock-time](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.mock-time) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145318256) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145331628) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145318676) [haskellPackages.nlopt-haskell](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.nlopt-haskell) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145324097) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145319109) [[:penguin::x:]](https://hydra.nixos.org/build/145317107) [haskellPackages.openai-servant](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.openai-servant) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/145319096) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145324152) [[:penguin::x:]](https://hydra.nixos.org/build/145328412) [haskellPackages.parsley-core](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.parsley-core) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145331668) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145319557) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145329267) [haskellPackages.picosat](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.picosat) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145330177) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145318136) [[:penguin::x:]](https://hydra.nixos.org/build/145335195) [haskellPackages.proto-lens](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.proto-lens) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145328885) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145323110) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145334892) [haskellPackages.ptr-poker](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.ptr-poker) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145320440) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145322568) [[:penguin::x:]](https://hydra.nixos.org/build/145327299) [haskellPackages.telegram-bot-simple](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.telegram-bot-simple) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145335322) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334700) [[:penguin::x:]](https://hydra.nixos.org/build/145330961) [haskellPackages.tophat](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.tophat) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145335836) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145316721) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145330104) [haskellPackages.type-natural](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.type-natural) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145323340) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145326261) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145324550) [haskellPackages.unicode-properties](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.unicode-properties) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145322563) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145330917) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145333850) [haskellPackages.wiringPi](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.wiringPi) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145321377) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335117) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145323155) [haskellPackages.x86-64bit](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.x86-64bit) 
- [ ] [[:iphone::x:]](https://hydra.nixos.org/build/145318340) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145317221) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145328164) [haskellPackages.zxcvbn-hs](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.zxcvbn-hs) 
</details>

#### Unmaintained packages with failed dependency
<details><summary>96 job(s) </summary>

- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145328843) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145331222) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145331062) [haskellPackages.Chart-cairo](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.Chart-cairo) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145328038) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333094) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145322916) [haskellPackages.Chart-gtk](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.Chart-gtk) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145320646) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145331217) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145330570) [haskellPackages.Chart-gtk3](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.Chart-gtk3) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145333156) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334963) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145328607) [haskellPackages.Chart-tests](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.Chart-tests) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145323253) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335017) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145322284) [haskellPackages.GtkTV](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.GtkTV) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145331423) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145332197) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145323732) [haskellPackages.H](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.H) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145325392) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327302) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145318730) [haskellPackages.MissingK](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.MissingK) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145320865) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335123) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145322817) [haskellPackages.PrimitiveArray](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.PrimitiveArray) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145331447) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334537) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145322481) [haskellPackages.aivika-experiment-cairo](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.aivika-experiment-cairo) 
- [ ] [bustle](https://hydra.nixos.org/eval/1678005?filter=bustle) 
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145333606) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145316308) [toplevel](https://hydra.nixos.org/eval/1678005?filter=bustle)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145317020) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145320455) [haskellPackages](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.bustle)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145320561) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333328) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145334942) [haskellPackages.cairo](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.cairo) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319826) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334344) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145317864) [haskellPackages.cairo-appbase](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.cairo-appbase) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319126) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145320334) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145330187) [haskellPackages.cairo-canvas](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.cairo-canvas) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145316904) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145321876) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145332186) [haskellPackages.chassis](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.chassis) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145324339) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327949) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145333462) [haskellPackages.compdoc](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.compdoc) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145325256) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145317945) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145318127) [haskellPackages.compdoc-dhall-decoder](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.compdoc-dhall-decoder) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145316863) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333001) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145327339) [haskellPackages.composite-aeson](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-aeson) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145316936) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145321916) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145321301) [haskellPackages.composite-aeson-cofree-list](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-aeson-cofree-list) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145334296) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145318503) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145333079) [haskellPackages.composite-aeson-path](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-aeson-path) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145323351) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145318793) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145335598) [haskellPackages.composite-aeson-refined](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-aeson-refined) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145330395) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335406) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145322003) [haskellPackages.composite-aeson-throw](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-aeson-throw) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145331363) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145315963) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319534) [haskellPackages.composite-aeson-writeonly](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-aeson-writeonly) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145321431) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145330932) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145329159) [haskellPackages.composite-binary](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-binary) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145324644) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145329605) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145320283) [haskellPackages.composite-ekg](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-ekg) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145330840) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145324831) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145328642) [haskellPackages.composite-hashable](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-hashable) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145321974) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145318241) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145315901) [haskellPackages.composite-swagger](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-swagger) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145318534) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333302) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145322105) [haskellPackages.composite-tuple](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-tuple) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145321853) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145325588) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145322611) [haskellPackages.composite-xstep](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.composite-xstep) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145332187) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145331044) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327856) [haskellPackages.diagrams-cairo](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.diagrams-cairo) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145334053) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145329822) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145335332) [haskellPackages.dynamic-graph](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.dynamic-graph) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145326013) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327413) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145331991) [haskellPackages.easytensor-vulkan](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.easytensor-vulkan) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145322032) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145324445) [haskellPackages.ghcjs-dom-hello](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.ghcjs-dom-hello) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145326420) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145332664) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145316180) [haskellPackages.gio](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gio) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145328411) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145323677) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145320274) [haskellPackages.glib](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.glib) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145329066) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145325725) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145329292) [haskellPackages.gloss-accelerate](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gloss-accelerate) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145318786) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334897) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145316016) [haskellPackages.goatee-gtk](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.goatee-gtk) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145331135) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145328643) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145315989) [haskellPackages.gtk](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145326849) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327026) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327627) [haskellPackages.gtk-helpers](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk-helpers) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145315831) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334156) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145329000) [haskellPackages.gtk-jsinput](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk-jsinput) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145327748) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145324402) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145329048) [haskellPackages.gtk-largeTreeStore](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk-largeTreeStore) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145320813) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145325564) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145321991) [haskellPackages.gtk-simple-list-view](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk-simple-list-view) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145323984) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145322638) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145331136) [haskellPackages.gtk-toggle-button-list](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk-toggle-button-list) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319441) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335596) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145329915) [haskellPackages.gtk-traymanager](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk-traymanager) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145321866) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145326783) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145331931) [haskellPackages.gtk3](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk3) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145317398) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145326479) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145320161) [haskellPackages.gtk3-helpers](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.gtk3-helpers) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145327843) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334220) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145317999) [haskellPackages.hXmixer](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.hXmixer) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145321116) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145317044) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145328775) [haskellPackages.happstack-static-routing](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.happstack-static-routing) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145331078) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145331258) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145322602) [haskellPackages.helm](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.helm) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145318379) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333260) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145320599) [haskellPackages.hierarchical-clustering-diagrams](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.hierarchical-clustering-diagrams) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145316458) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145325169) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145330600) [haskellPackages.hmatrix-nlopt](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.hmatrix-nlopt) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145324867) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145331340) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145317207) [haskellPackages.ihaskell-charts](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.ihaskell-charts) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145328032) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145325077) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145331211) [haskellPackages.ihaskell-diagrams](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.ihaskell-diagrams) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145318818) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145328238) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145318405) [haskellPackages.ihaskell-inline-r](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.ihaskell-inline-r) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145323286) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145317455) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145321979) [haskellPackages.ihaskell-plot](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.ihaskell-plot) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145328253) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145332845) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145334826) [haskellPackages.indian-language-font-converter](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.indian-language-font-converter) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145324940) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333710) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145330559) [haskellPackages.isiz](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.isiz) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319828) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145328198) [haskellPackages.jsaddle-webkit2gtk](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.jsaddle-webkit2gtk) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145317903) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145325705) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145324325) [haskellPackages.keera-hails-mvc-solutions-config](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.keera-hails-mvc-solutions-config) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145335965) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145317314) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145329654) [haskellPackages.keera-hails-mvc-view-gtk](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.keera-hails-mvc-view-gtk) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145332529) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145319382) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327080) [haskellPackages.libnotify](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.libnotify) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145326173) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333500) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145318860) [haskellPackages.nc-indicators](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.nc-indicators) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145318498) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145331439) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145333075) [haskellPackages.opentelemetry-extra](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.opentelemetry-extra) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145327366) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327936) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327303) [haskellPackages.opentelemetry-lightstep](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.opentelemetry-lightstep) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145323135) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145323708) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145322216) [haskellPackages.osdkeys](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.osdkeys) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145321815) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334480) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145333235) [haskellPackages.pango](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.pango) 
- [ ] [[:iphone::heavy_check_mark:]](https://hydra.nixos.org/build/145321142) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145322837) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145333769) [haskellPackages.parsley-garnish](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.parsley-garnish) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145317332) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145325708) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145321528) [haskellPackages.plot](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.plot) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145327336) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145324082) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145324044) [haskellPackages.profiterole](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.profiterole) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145323866) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145316941) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145324692) [haskellPackages.profiteur](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.profiteur) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145322939) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334003) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145322738) [haskellPackages.proto-lens-jsonpb](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.proto-lens-jsonpb) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319478) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335367) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145327307) [haskellPackages.proto-lens-protobuf-types](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.proto-lens-protobuf-types) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145320118) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327918) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145325592) [haskellPackages.proto-lens-protoc](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.proto-lens-protoc) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145331977) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145332205) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145320350) [haskellPackages.proto-lens-runtime](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.proto-lens-runtime) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319551) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145317143) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145320364) [haskellPackages.proto-lens-setup](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.proto-lens-setup) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145316561) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145329095) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145323752) [haskellPackages.qr](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.qr) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145331819) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145322667) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145325693) [haskellPackages.recursive-line-count](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.recursive-line-count) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145324732) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334270) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145316621) [haskellPackages.rounded](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.rounded) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145331022) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145318190) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145321446) [haskellPackages.sdl2-cairo](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.sdl2-cairo) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145320140) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145328388) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319497) [haskellPackages.signable-haskell-protoc](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.signable-haskell-protoc) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145326940) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145325867) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327532) [haskellPackages.sized](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.sized) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145332213) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145327005) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145323682) [haskellPackages.slope-field](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.slope-field) 
- [ ] [splot](https://hydra.nixos.org/eval/1678005?filter=splot) 
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145329446) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145316448) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327611) [toplevel](https://hydra.nixos.org/eval/1678005?filter=splot)
  - [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319025) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333985) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145325573) [haskellPackages](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.splot)
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145323098) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145328402) [[:penguin::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145329738) [haskellPackages.tensorflow-proto](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.tensorflow-proto) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145332319) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335050) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145333622) [haskellPackages.threadscope](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.threadscope) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319991) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145324785) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145320890) [haskellPackages.timeplot](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.timeplot) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145319464) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333665) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145335755) [haskellPackages.unicode-names](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.unicode-names) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145332730) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145318096) [haskellPackages.webkit2gtk3-javascriptcore](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.webkit2gtk3-javascriptcore) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145334168) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145329901) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145315972) [haskellPackages.wholepixels](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.wholepixels) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145332634) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145334859) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145335068) [haskellPackages.xdot](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.xdot) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145331081) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145335426) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327091) [haskellPackages.xmonad-screenshot](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.xmonad-screenshot) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145329600) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145333151) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327921) [haskellPackages.yesod-media-simple](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.yesod-media-simple) 
- [ ] [[:iphone::heavy_exclamation_mark:]](https://hydra.nixos.org/build/145321388) [[:apple::hourglass_flowing_sand:]](https://hydra.nixos.org/build/145317631) [[:penguin::heavy_check_mark:]](https://hydra.nixos.org/build/145327325) [haskellPackages.yi-frontend-pango](https://hydra.nixos.org/eval/1678005?filter=haskellPackages.yi-frontend-pango) 
</details>

*Report generated with [maintainers/scripts/haskell/hydra-report.hs](https://github.com/NixOS/nixpkgs/blob/haskell-updates/maintainers/scripts/haskell/hydra-report.sh)*

cdepillabout committed:
spago: set earlier dhall dependency

expipiplus1 commented:
upstream issue for hnix: https://github.com/haskell-nix/hnix/issues/952

sternenseemann committed:
haskellPackages.hnix-store-{core,remote}: downgrade to < 0.5

hnix 0.13.* doesn't support hnix-store-* >= 0.5 yet, pending some
refactors to get GHC 9.0.x support working. Until that happens,
we downgrade hnix-store-* since nothing needs the new version yet.

https://github.com/haskell-nix/hnix/issues/952

sternenseemann commented:
Potential fix for hnix <https://github.com/NixOS/nixpkgs/pull/126840>, should unbreak all remaining maintained packages as far as I can see

maralorn committed:
Merge pull request #126840 from sternenseemann/hnix-store-downgrade

haskellPackages.hnix-store-{core,remote}: downgrade to < 0.5

sternenseemann committed:
haskellPackages.wide-word: remove now unnecessary override

sternenseemann committed:
haskellPackages.dhall-{openapi,nix}: unrestrict

stackage has updated to dhall 1.39, so we can update these as well:

haskellPackages.dhall-openapi: 1.0.0 -> 1.0.1

haskellPackages.dhall-nix: 1.1.20 -> 1.1.21

cdepillabout committed:
haskellPackages: update documentation about Darwin builds

maralorn committed:
haskellPackages.hls-brittany-plugin: dontCheck on darwin

maralorn committed:
Merge branch 'master' into haskell-updates

tbidne committed:
haskellPackages.xmonad-wallpaper: jailbreak

Compiles with random 1.2.0, but bound is too strict at <1.2.

Co-authored-by: sterni <sternenseemann@systemli.org>

expipiplus1 committed:
haskellPackages.shellFor: Work around hoogle generation failure

Workaround for https://github.com/NixOS/nixpkgs/issues/82245

Although this doesn't tackle the root cause of a null package sneaking
in (via executableHaskellDepends), it does effectively treat the symptom
by just ignoring any null packages.

Seeing as that issue has been open for more than a year I think this
band-aid is necessary.

expipiplus1 committed:
nix-linter: 2020-09-25 -> 2021-06-16

Also add to haskellPackages

cdepillabout committed:
Merge pull request #127028 from expipiplus1/joe-hoogle-fix

haskellPackages.shellFor: Work around hoogle generation failure

sternenseemann committed:
haskellPackages.errata: no longer mark as broken

As pointed out here [1], errata has started working on its own again.

[1]: https://github.com/hyperrealgopher/burrow/pull/27#issue-670953936

sternenseemann committed:
haskellPackages.table-layout: unbreak

cdepillabout committed:
Merge pull request #126903 from cdepillabout/hacking.md-for-darwin

haskellPackages: update documentation about Darwin builds

sternenseemann committed:
haskellPackages.cabal2nix-unstable: wrap with runtime dependencies

sternenseemann committed:
haskellPackages.composite-{base,aeson}: unbreak

sternenseemann committed:
haskellPackages.proto-lens: unbreak

sternenseemann committed:
haskellPackages.tophat: unbreak

maralorn committed:
Merge branch 'master' into haskell-updates

maralorn committed:
haskellPackages: mark builds failing on hydra as broken

This commit has been generated by maintainers/scripts/haskell/mark-broken.sh
